plugins/twind.ts

```diff
@@ -15,9 +15,9 @@ export default function(state) { hydrate(options, state); }`;
   return {
     name: "twind",
     entrypoints: { "main": main },
-    render(ctx) {
+    async renderAsync(ctx) {
       sheet.reset(undefined);
-      const res = ctx.render();
+      const res = await ctx.renderAsync();
```

Comment:
This doesn't work. Twind relies on thread local state (like context or hooks). Try rendering two pages in parallel

Reply:
That's a good point, I'll take a look.